presto-hive/src/main/java/com/facebook/presto/hive/HiveColumnHandle.java

```diff
@@ -52,11 +52,16 @@
     public static final HiveType BUCKET_HIVE_TYPE = HIVE_INT;
     public static final TypeSignature BUCKET_TYPE_SIGNATURE = BUCKET_HIVE_TYPE.getTypeSignature();
 
-    public static final int FILE_SIZE_COLUMN_INDEX = -13;
```

Comment:
Looks like this fix to avoid the accidental column index collision should be in the other commit so that we don't accidentally introduce a bug if we revert the modified time changes but not the file size changes (for whatever reason).

Reply:
@pettyjamesm I have placed the change in the correct commit.